apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -115,3 +115,32 @@ function populate_wp_template_data() {
 	$fse->insert_default_data();
 }
 register_activation_hook( __FILE__, __NAMESPACE__ . '\populate_wp_template_data' );
+
+/**
+ * Add front-end CoBlocks gallery block scripts.
+ *
+ * This function performs the same enqueueing duties as `CoBlocks_Block_Assets::frontend_scripts`,
+ * but for our FSE header and footer content. `frontend_scripts` uses `has_block` to determine
+ * if gallery blocks are present, and `has_block` is not aware of content sections outside of
+ * post_content yet.
+ */
+function enqueue_coblocks_gallery_scripts() {
+	if ( ! is_plugin_active( 'coblocks' ) ) {
+		return;
+	}
+
+	$template = new WP_Template();
+	$header   = $template->get_template_content( 'header' );
+	$footer   = $template->get_template_content( 'footer' );
+
+	if ( has_block( 'coblocks/gallery-masonry', $header . $footer ) ) {
```

Comment:
Any reasons why we are only checking for `/gallery-masonry` and not for `gallery-carousel` as well?

Reply:
Good point! Probably @kwight didn't included it because the gallery carousel block is not supported in Simple sites. 

But it is available when using the CoBlocks plugin in an AT site, so we should also load those styles when the block is found.

Reply:
Added in 82c7205045a5bc41fff5d6c46b3d77bfa11668a7